### ISSUE TITLE
Assets path

### ISSUE THREAD

**Issue by afercia** [Yoast/wpseo-news#203]
Noticed on trunk:
```
Failed to load resource: the server responded with a status of 404 (Not Found)
http://.../wp-content/plugins/wpseo-news-trunk/classes/assets/post-edit.js?ver=3.0
```
and
```
GET http://.../wp-content/plugins/wpseo-news-trunk/classes/assets/admin-page.js?ver=3.0 404 (Not Found)
```
See `WPSEO_News::get_file()` which is in a file in a subdirectory and it is used as parameter for `plugin_url()` when enqueuing the assets. Should maybe use `dirname( __FILE__ )`

**Comment by afercia** [comment]
Noticed a third use of `WPSEO_News::get_file()` in  `WPSEO_News_Product` so I guess it's meant to return the "slug" of the main plugin file ?

**Comment by afercia** [comment]
As far as I see this was fixed in the 3.1 release branch and now `get_file()` is deprecated in favor of `WPSEO_NEWS_FILE`.
